### DISCUSSION TITLE
添加迅雷云盘只能上传删除，没法下载

### DISCUSSION THREAD

**Discussion by acegao9527** [AlistGo/alist#1141] [Q&A]
我试了两个版本：最新版和5.2.1，分别独立安装的Docker都是这个问题，请问可能是什么原因？

而且保存完，进配置后台看到verified captcha token还是没值，直接加载data.db,把verified captcha token手动刷进库了，再打开配置后台就有值了。但是还是只能上传不能下载，是迅雷做了什么限制吗？

**Comment by luvjojo** [comment]
我也是迅雷云盘出了问题，无法下载，无法在线观看视频，配置都是按教程来的，也挂载成功了，能看见文件，能删除。

**Comment by wuyuzheyan** [comment]
我也是一样，可上传可删除，就是不能下载和播放视频文件

**Comment by WangEdward** [comment]
需要在下载请求加入客户端的 UA 头

**Reply by Cyiming** [reply]
> 需要在下载请求加入客户端的 UA 头

好像不是 在线播放不涉及到ua头吧
